src/docs/goals/careers.md

```diff
@@ -7,34 +7,16 @@ featured: ../images/featured/goals.png
 
 Building meaningful career ladders for individual contributors can be a real challenge. One of the ways to support someone's career is to give them multiple role archetypes to choose from, and letting them play on their strengths. One of the goals of Auxiliary Engineering is to give senior engineers a new archetype which feels challenging, rewarding, and well aligned with their long term career success and ambitions.
 
-## Influencing without Authority or Dependency
-
-## WORK IN PROGRESS NOTES
-
-- On Dependency
-  - When someone has unique strength in an area, it's too easy to lean on that person
-  - In the short term, dependency can satisfy one's ego, and it's generally the path of least resistance
-  - In the long run, dependency feels bad, and it also bakes a lot of risk into a team
-  - In a 3 month engagement, the end of the project is always looming on the horizon
-  - This makes it easy to challenge knowledge dependency early on, and prevent single points of knowledge failure from forming
-  - Expectations
-
-- Breadth of influence across the organization
-- Regularly shaking up their problem domain
-- Being in a position to identify the meta opportunities and having autonomy to drive innovation
-
-- Senior talent enjoys working on a variety of projects
-- Similar to high-end consultant
-  - Feel more connected to results than a consultant
-  - But it's not a 0 sum game like consulting
-  - There's no incentive to build dependency.
-- Building strength without dependency
-  - There's an explicit goal to NOT build dependency
-  - Healthy knowledge sharing
-  - Feels impactful. One sees the results first hand
-- Large business impact
-  - Moves some of an organizations most talented engineers around the company
-  - Able to influence a large number of projects in healthy ways
-    - Influencing as part of a team, not an outsider disconnected from implementation / results
-  - Avoids "powerpoint" architects. Keeps principal engineers writing code
-    - This helps keep them marketable. It's good for their career.
+## Diversity by Design
+
+Providing a fresh problem space with some familiarity in how to solve it can be refreshing for individual contributors. By providing value in many areas through their direct involvement, a shared goal in Aux Eng lies in providing technologists a meaningful breadth of influence. That influence comes from understanding a many of the diverse problems in a software ecosystem. Especially in senior technical tracks and leadership roles, that kind of exposure and influence in an organization goes a long way in building a career.
```

Comment:
```suggestion
Providing a fresh problem space with some familiarity on how to solve it can be refreshing for individual contributors. By delivering value in many areas through their direct involvement, a shared goal in Aux Eng lies in providing technologists with a meaningful degree of influence. That influence comes from understanding many of the diverse problems in a software ecosystem. Especially in senior technical tracks and leadership roles, this kind of exposure and influence in an organization goes a long way in building a career.
```